internal/printer/printer_test.go

```diff
@@ -283,20 +297,33 @@ const someProps = {
 				Scope:       "astro-XXXX",
 				InternalURL: "http://localhost:3000/",
 			})
-			output := string(result.Output)
+			output := strings.TrimSpace(test_utils.Dedent(string(result.Output)))
```

Comment:
Also just ran Dedent on everything because I was tired of tabs vs spaces diffs. Hopefully we won’t see too much of that again